test/test_jit.py

```diff
@@ -351,7 +351,7 @@ def allSum(vs):
                 vs = vs[:-drop]
             # we don't want all the grad for all the outputs to be the same
             # so we multiply each by a constant
-            return sum([(i + 1) * v.sum() for i, v in enumerate(vs) if v is not None])
+            return sum([math.log(i + 2) * v.sum() for i, v in enumerate(vs) if v is not None])
```

Comment:
I'm not sure what is going on here: why did this need to change?

Reply:
Because `(i + 1)` became really large when we had tensors with thousands of outputs, and that caused the second order gradients to be in the order of tens of thousands at least. That's not a good regime for floating-point calculations, so I replaced it with a much slower growing `log`.